tensorflow/core/kernels/data/experimental/unbatch_dataset_op.cc

```diff
@@ -222,6 +239,7 @@ class UnbatchDatasetOp : public UnaryDatasetOpKernel {
 
     const DatasetBase* const input_;
     std::vector<PartialTensorShape> shapes_;
+    int64 known_batch_size_;
```

Comment:
can we call this just `batch_size_`? Then add a comment that it may or may not be known, with -1 representing unknown.

Reply:
Thanks @aaudiber, updated.